### ISSUE TITLE
Open air fusion not reliable anymore

### ISSUE THREAD

**Issue by Ghilker** [tgstation/tgstation#47156]
Issue reported from  Round ID: 121609 (/tg/Station Terry [EU] [100% LAG FREE] [New-Dedi])

Reporting client version: 512.1488

![Cattura](https://user-images.githubusercontent.com/42839747/67004356-6d622580-f0e0-11e9-9108-7c1a7a405494.PNG)
After the bug patch that removed the space heating from fusion, i started testing the open air fusion as a mean to make a reactor to power the station by adding collector arrays near the glass (not yet built in the picture)
Before the particles kept shooting, hitting the collectors and making tens of MW (reaching 50 MW at peack)
Now is hard to start, impossible to maintain (the two pipes on the left are connected to injectors that constantly inject the fusion mix) and is not worth anymore the throuble

**Comment by FloranOtten** [comment]
Are you sure it is because of the space heating patch? Considering the timing,  it'd seem more likely  caused by #47073

**Comment by Ghilker** [comment]
No I don't mean is the space heating patch, in fact thank to that I could do the setup to make open air fusion

I'm sure is because of #47073

**Comment by kriskog** [comment]
If the instability is high, you will be overall more exothermic and the reaction will lose heat regardless of the 1e8 ceiling. If your reaction is failing because of the ceiling at 1e8 you got some serious issues in your setup and mix, causing you to lose literally 100 million C without gaining any back.

tl;dr You'll need to adjust your injection and/or mix to be less unstable.

**Comment by Ghilker** [comment]
No, because i've done this lots of time, kickstarting the reaction with a 5e6 k canister and then letting it do its thing. 
Same mix, same amount of tritium going in. Now it starts, shoot particles for 30 seconds and it cools off. 
Is not because my mix is unstable, because it starts the reaction easily and mantain it at start, after it reaches 1e8 k it slows down untill full stop. 

Also since this affects the recent change to 1e8 cap, Hypernobilium is harder to make, witch is good, because i was able to make 100 moles of hyper by adding more or less 2000 moles of BZ

**Comment by kriskog** [comment]
I suggest you try running it where you got a gas sensor so you can see whats going on. If the heat is stable at 1e8 (never goes below) you won't see particles even if the fusion is still technically going as those require fusion power that the above the ceiling reactions simply dont have.

As an aside I did notice while looking at the code again that stable reactions aren't heat capped. You might have ended up heating it far past the 1e8 max, causing a long string of neutral reactions.

**Comment by kriskog** [comment]
Allright, I did a new PR. #47161
It basically makes fusion behave just like before, just not letting it get hot. Will close this.
